### DISCUSSION TITLE
Welcome to radolan2map Discussions!

### DISCUSSION THREAD

**Discussion by Weathermann** [Weathermann/radolan2map#3] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jamaa** [comment]
Hi, I am an engineer working at @sydroconsult, where we use REGNIE data quite extensively (and sometimes also RADOLAN). I am currently working on adding REGNIE to raster (tif) conversion functionality to radolan2map (#2). 

I would like to explore ways to extract and display REGNIE (and RADOLAN) data in the form of time series within QGIS. E.g. having raster layers with timestamps which you can animate, or allowing the user to select a point or an area on the map, for which you can then display a time series.
